tedana/tests/test_integration.py

```diff
@@ -277,6 +278,44 @@ def test_integration_five_echo(skip_integration):
     check_integration_outputs(fn, out_dir)
 
 
+def test_integration_robustica_five_echo(skip_integration):
```

Comment:
The integration tests are the slowest tests. Our goal for testing is to aim for every line of code to be run somewhere in the tests. You've added integration tests for all three datasets. Are they testing different sections of the new robust ICA code or can we just run an integration test using robustICA on only one dataset? We might also be able to just swap one of the 3 datasets so that two are run using fastICA and one is run using robustICA.

The above is particularly true since these integration tests aren't checking if the outputted values are consistent, just that they fully run. If it might be useful to test the consistently of robust ICA related functions, that might be better down with tests of specific functions rather than through an integration test.

Reply:
For an integration test or function-specific test, is there a way to make sure DBSCAN doesn't converge and the (current) exception to run AgglomerativeClustering is run instead?